spec/system/public/cclow/climate_targets_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe 'Climate Targets search', type: 'system' do
+  before do
+    visit 'cclow/climate_targets'
+  end
+
+  it 'loads the page' do
+    expect(page).to have_text('All Climate Targets')
+    expect(page).to have_text('Showing 1993 results')
```

Comment:
These tests would need to be updated if we change the test dump, right? Not sure if I'm understanding correctly.

Reply:
Yes, that's why it's not that great. Although, seeds were not changed for a quite a while